be/test/io/cache/block_file_cache_test.cpp

```diff
@@ -7913,4 +7913,81 @@ TEST_F(BlockFileCacheTest, cached_remote_file_reader_normal_index) {
     FileCacheFactory::instance()->_capacity = 0;
 }
 
+TEST_F(BlockFileCacheTest, test_reset_capacity) {
```

Comment:
should we also test `validate_capacity()`?

Reply:
validate_capacity is already tested in this case